examples/ml_perf/main.py

```diff
@@ -0,0 +1,246 @@
+import argparse
+import importlib
+import os
+
+os.environ["KERAS_BACKEND"] = "jax"
+
+import keras
+
+import keras_rs
+
+from .dataloader import DataLoader
+from .model import DLRMDCNV2
+
+SEED = 1337
+
+
+def main(
+    file_pattern,
+    dense_features,
+    large_emb_features,
+    small_emb_features,
+    label,
+    shuffle_buffer,
+    embedding_dim,
+    allow_id_dropping,
+    max_ids_per_partition,
+    max_unique_ids_per_partition,
+    embedding_learning_rate,
+    bottom_mlp_dims,
+    top_mlp_dims,
+    num_dcn_layers,
+    dcn_projection_dim,
+    learning_rate,
+    global_batch_size,
+    num_epochs,
+):
+    # Set DDP as Keras distribution strategy
+    devices = keras.distribution.list_devices(device_type="tpu")
+    distribution = keras.distribution.DataParallel(devices=devices)
+    keras.distribution.set_distribution(distribution)
+    num_processes = distribution._num_process
+
+    per_host_batch_size = global_batch_size // num_processes
+
+    # === Distributed embeddings' configs for sparse features ===
+    feature_configs = {}
+    for large_emb_feature in large_emb_features:
+        # Rename these features to something shorter; was facing some weird
+        # issues with the longer names.
+        feature_name = (
+            large_emb_feature["name"]
+            .replace("-", "_")
+            .replace("egorical_feature", "")
+        )
+        vocabulary_size = large_emb_feature["vocabulary_size"]
+        multi_hot_size = large_emb_feature["multi_hot_size"]
+
+        table_config = keras_rs.layers.TableConfig(
+            name=f"{feature_name}_table",
+            vocabulary_size=vocabulary_size,
+            embedding_dim=embedding_dim,
+            # TODO(abheesht): Verify.
+            initializer=keras.initializers.VarianceScaling(
```

Comment:
The default is `keras.initializers.VarianceScaling(mode="fan_out")`, which has a scale of 1 and a uniform distribution.

Why did you need to change it?